test/module-test.cc

```diff
@@ -121,10 +127,23 @@ TEST(module_test, format_to) {
   fmt::memory_buffer mb;
   fmt::format_to(mb, "{}", 42);
   EXPECT_EQ("42", std::string_view(buffer));
+
+  std::wstring w;
+  fmt::format_to(std::back_inserter(w), L"{}", 42);
+  EXPECT_EQ(L"42", w);
+
+  wchar_t wbuffer[4] = {0};
+  fmt::format_to(wbuffer, L"{}", 42);
+  EXPECT_EQ(L"42", std::wstring_view(wbuffer));
+
+  fmt::wmemory_buffer wb;
+  fmt::format_to(wb, L"{}", 42);
+  EXPECT_EQ(L"42", std::wstring_view(wbuffer));
```

Comment:
As before, I think this does too much functional testing. I suggest having a single wide format_to call.

Reply:
These are different overloads in the API. I think this is the minimal set for full coverage.